greenscraper/exclusions.txt

```diff
@@ -0,0 +1,41 @@
+Greene
+Greenley
+Greenwald
+Greenberg
+Blue/Green
+blue/green
+Green Light
+Blue-green
+Blue green
+COVIDGreen
+Greenwood
+Greentree
+Gvergreen
+Greenfield
+Greenleaf
+Greenhill
+Greenway
+Greenland
+Greenlee
+Greenville
+Greenwood
+Greenberg
+Greenfield
+Greenbaum
+Greenberg
+Greenwald
+Greenspan
+Greene
+Greenberg
+Greenman
+Greenland
+Greenleaf
+Greenlee
+Greenspan
+Greenstein
+Greenwald
+Greenwell
+Greenblatt
+Greenberg
+Greenfield
+Greenbaum
```

Comment:
I am wondering if we can just match on `" Green "` so anything that has green as part of a word does not get matched

Reply:
The issue is that there are several words that contain "green" and are relevant to the search/topic

Reply:
I could have it as a "TODO" to explore optimization in the future?

Reply:
sounds good